src/routes/(app)/store/[productId]/+page.svelte

```
@@ -141,8 +141,9 @@
               <div style="transform: translateX(165px) translateX(-50%); transition-duration: 0.3s;"></div>
             </div>
           </div>
-
-          {@html product.faqs}
+          <div class="!text-white">
+            {@html product.faqs}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)